torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -960,6 +969,20 @@ def intify(x):
             else:
                 return x
 
+        for idx, arg_name in enumerate(variable.kernel.arg_names):
+            if idx in variable.kernel.constexprs:
+                assert arg_name in combined_args_raw
+                if isinstance(combined_args_raw[arg_name], SymNodeVariable):
+                    # This arg is marked as tl.constexpr. That means that triton will recompile every time
+                    # this value changes.
+                    # https://github.com/pytorch/pytorch/issues/136504
+                    # One option is to correctly pass the symints in so that the symbolic expressions are defined
+                    # when the triton code is being executed.
+                    # But since triton will have to recompile either way, we instead just specialize on the value.
+                    combined_args_raw[arg_name] = ConstantVariable.create(
```

Comment:
instead of specializing here why dont we just specialize the value of s0 at inductor time similar to how to handle other dynamic shape for user defined triton kernels?